scripts/release-deb.sh

```diff
@@ -228,37 +247,33 @@ RUN apt-get -qq -y --no-install-recommends install \
     libdistro-info-perl \
     pandoc \
     rsync \
-    sensible-utils && \
+    sensible-utils \
+    sudo && \
   apt-get clean && \
   rm -rf /tmp/* /var/tmp/*
 
 COPY --from=golang /usr/local/go /usr/local/go
 
 # build debian package as yq user
```

Comment:
Is that really necessary? Why not just build it as an existing user, such as 'nobody'?

Reply:
That was coded by previous author so I can't answer why. But my guess is that the script needs write permission in mapped folder, so you cannot use nobody.